tests/empty_struct.rs

```diff
@@ -0,0 +1,15 @@
+#[macro_use] extern crate custom_derive;
+#[macro_use] extern crate derive_builder;
+
+custom_derive!{
+    /// This is a doc comment for the struct
+    #[warn(missing_docs)]
+    #[allow(non_snake_case)]
+    #[derive(Debug, PartialEq, Default, Builder)]
+    struct IgnoreEmptyStruct {  }
```

Comment:
If we do want to support that, the bigger picture might be "don't fail on anything we are (a) not interested in and which is (b) parse-able by custom_derive". That would also include tuple-structs and enums. If you like that idea, we can open a low-priority tracking issue for that. :-)